assets/inflection.yaml

```diff
@@ -45,3 +45,28 @@
   monthly_active_users: ''
   user_distribution: ''
   failures: ''
+
+- type: model
+  name: Inflection-2
+  organization: Inflection AI
+  description: Inflection-2 is the best model in the world for its compute class and the second most capable LLM in the world, according to benchmark evaluation, as of its release.
+  created_date: 2023-11-22
+  url: https://inflection.ai/inflection-2
+  model_card: none
+  modality: text; text
+  analysis: Evaluated against state of the art models on benchmarks, and found to be most performant model outside of GPT-4.
+  size:
+    value: unknown
+    explanation: Likely larger than Inflection-1, which was stated to be around the size of ChatGPT (175B parameters).
```

Comment:
Where is this from? I don't think this is true if I were to guess

Reply:
I might be misunderstanding, but in both Inflection 2's release doc it's stated to be in the same compute class as PaLM 2 and Inflection 1's release doc is said to be in the same compute class as GPT-3.5. Am I conflating compute class and model size?